sdk/containerapps/Azure.ResourceManager.AppContainers/src/Generated/Models/ContainerAppJobTriggerType.cs

```diff
@@ -22,12 +22,12 @@ public ContainerAppJobTriggerType(string value)
             _value = value ?? throw new ArgumentNullException(nameof(value));
         }
 
-        private const string ScheduledValue = "Scheduled";
+        private const string ScheduleValue = "Schedule";
         private const string EventValue = "Event";
         private const string ManualValue = "Manual";
 
-        /// <summary> Scheduled. </summary>
-        public static ContainerAppJobTriggerType Scheduled { get; } = new ContainerAppJobTriggerType(ScheduledValue);
```

Comment:
breaking change, an enum value is renamed

Reply:
The enum was changed to have the correct value, we discussed and got this breaking change approved:
https://github.com/Azure/azure-rest-api-specs/pull/24400